### DISCUSSION TITLE
auto restart the scanner when a device is rotated from landscape to portrait or otherway arround

### DISCUSSION THREAD

**Discussion by RobEbbers** [mebjas/html5-qrcode#682] [Q&A]
good morning i am looking at how i could make the scanner auto restart when the screen is rotated to stop strange issues accuring with the scanner image like :
![afbeelding](https://user-images.githubusercontent.com/75092300/214017664-2272ebc3-ad24-42b4-a34e-8796f5882852.png)

**Comment by ROBERT-MCDOWELL** [comment]
meanhilw you can disable autorotate like disableFlip: true before it's fixed.
